picard/ui/metadatabox/__init__.py

```diff
@@ -647,6 +640,12 @@ def _update_items(self, result=None, error=None):
         readonly_item_flags = QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled
         editable_item_flags = readonly_item_flags | QtCore.Qt.ItemFlag.ItemIsEditable
         alignment = QtCore.Qt.AlignmentFlag.AlignLeft | QtCore.Qt.AlignmentFlag.AlignTop
+        colors = {
+            TagStatus.UNCHANGED: self.palette().color(QtGui.QPalette.ColorRole.Text),
+            TagStatus.REMOVED: QtGui.QBrush(interface_colors.get_qcolor('tagstatus_removed')),
+            TagStatus.ADDED: QtGui.QBrush(interface_colors.get_qcolor('tagstatus_added')),
+            TagStatus.CHANGED: QtGui.QBrush(interface_colors.get_qcolor('tagstatus_changed'))
+        }
```

Comment:
Setting up the colors list looks like quite some work. Given that the update of the metadata is a performance critical part, wouldn't it be better to keep this outside the function?

If we need to react on config changes to use the new colors we could have `InterfaceColors` emit a signal whenever colors change, and MetadataBox could update the colors (and even refresh the view).